src/Dotnet.Script.Tests/ScriptExecutionTests.cs

```diff
@@ -470,7 +479,7 @@ private static string CreateTestScript(string scriptFolder)
 
         private static void CreateTestPackage(string packageLibraryFolder)
         {
-            ProcessHelper.RunAndCaptureOutput("dotnet", "new classlib -n NuGetConfigTestLibrary", packageLibraryFolder);
+            ProcessHelper.RunAndCaptureOutput("dotnet", "new classlib -n NuGetConfigTestLibrary -f netstandard2.0", packageLibraryFolder);
```

Comment:
This is needed if you have the .Net 5 SDK installed since that version no longer defaults to `netstandard2.0`